scripts/create_changelog.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
we have an updated versions of this file with more comments. Let's use this instead: https://github.com/kyma-project/nats-manager/blob/main/.github/scripts/create_changelog.sh

Reply:
Done